src/server/scripts/World/npcs_special.cpp

```diff
@@ -2528,6 +2528,217 @@ class npc_train_wrecker : public CreatureScript
         }
 };
 
+/*######
+## npc_argent_squire/gruntling
+######*/
+
+enum Misc
+{
+    STATE_BANK      = 0x1,
+    STATE_SHOP      = 0x2,
+    STATE_MAIL      = 0x4,
+
+    GOSSIP_ACTION_MAIL = 3,
+    ACHI_PONY_UP                = 3736,
+
+    SPELL_CHECK_MOUNT           = 67039,
+    SPELL_CHECK_TIRED           = 67334,
+    SPELL_SQUIRE_BANK_ERRAND    = 67368,
+    SPELL_SQUIRE_POSTMAN        = 67376,
+    SPELL_SQUIRE_SHOP           = 67377,
+    SPELL_SQUIRE_TIRED          = 67401
+};
+
+enum Quests
+{
+    QUEST_CHAMP_ORGRIMMAR       = 13726,
+    QUEST_CHAMP_UNDERCITY       = 13729,
+    QUEST_CHAMP_SENJIN          = 13727,
+    QUEST_CHAMP_SILVERMOON      = 13731,
+    QUEST_CHAMP_THUNDERBLUFF    = 13728,
+    QUEST_CHAMP_STORMWIND       = 13699,
+    QUEST_CHAMP_IRONFORGE       = 13713,
+    QUEST_CHAMP_GNOMEREGAN      = 13723,
+    QUEST_CHAMP_DARNASSUS       = 13725,
+    QUEST_CHAMP_EXODAR          = 13724
+};
+
+enum Pennants
+{
+    SPELL_DARNASSUS_PENNANT     = 63443,
+    SPELL_EXODAR_PENNANT        = 63439,
+    SPELL_GNOMEREGAN_PENNANT    = 63442,
+    SPELL_IRONFORGE_PENNANT     = 63440,
+    SPELL_ORGRIMMAR_PENNANT     = 63444,
+    SPELL_SENJIN_PENNANT        = 63446,
+    SPELL_SILVERMOON_PENNANT    = 63438,
+    SPELL_STORMWIND_PENNANT     = 62727,
+    SPELL_THUNDERBLUFF_PENNANT  = 63445,
+    SPELL_UNDERCITY_PENNANT     = 63441
+};
+
+class npc_argent_squire : public CreatureScript
+{
+public:
+    npc_argent_squire() : CreatureScript("npc_argent_squire") { }
+
+    bool OnGossipHello(Player* pPlayer, Creature* pCreature) override
+    {
+        // Argent Pony Bridle options
+        const AchievementEntry * = sAchievementStore.LookupEntry(ACHI_PONY_UP);
```

Comment:
```
/home/travis/build/TrinityCore/TrinityCore/src/server/scripts/World/npcs_special.cpp:2588:34: fatal error: 
      expected unqualified-id
        const AchievementEntry * = sAchievementStore.LookupEntry(ACHI_PONY_UP);
```
Follow the code standard instead:
```cpp
    AchievementEntry const* achievementEntry = sAchievementStore.LookupEntry(achievementId))
```
So you could probably use something like this:
```cpp
        AchievementEntry const* achievementEntry = sAchievementStore.LookupEntry(ACHI_PONY_UP);
```